# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/newtab/AboutNewTabService.jsm
# Commit: 7ec776d31a2c
# Full Hash: 7ec776d31a2c6e0ffc1e5ee311d63993317ea7ca
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2020-04-17 21:46:50
# Regressor Bug: 1614502
# File Overlap Count: 1
# Description:
#   Bug 1614502 - Expose utility functions for rendering Activity Stream. r=Mardak
#   
#   These will be used by a later patch inside of a ChromeWorker using
#   ReactDOMServer.
#   
# ==============================================================================

diff -r 8bf3fe111c40 -r 7ec776d31a2c browser/components/newtab/AboutNewTabService.jsm
--- a/browser/components/newtab/AboutNewTabService.jsm	Fri Apr 17 15:31:06 2020 +0000
+++ b/browser/components/newtab/AboutNewTabService.jsm	Fri Apr 17 15:31:11 2020 +0000
@@ -359,6 +359,7 @@
             `${BASE_URL}vendor/redux.js`,
             `${BASE_URL}vendor/react-redux.js`,
             `${BASE_URL}data/content/activity-stream.bundle.js`,
+            `${BASE_URL}data/content/newtab-render.js`,
           ];
 
           for (let script of scripts) {